policyengine_us/parameters/gov/hhs/medicaid/eligibility/immigration.yaml

```
@@ -0,0 +1,118 @@
+description: The following states cover undocumented immigrats under the medicaid program.
+metadata:
+  unit: bool
+  period: year
+  label: Medicaid state immigration status eligibility
+  breakdown:
+  - state_code
+  reference:
+    - title: "Medicaid Financial Eligibility for Seniors and People with Disabilities: Findings from a 50-State Survey | KFF"
+      href: https://files.kff.org/attachment/Table-3-Medicaid-and-CHIP-Eligibility-and-Enrollment-Policies-as-of-January-2021.pdf
```

That link didn't work - not sure if this is the right one though? I searched Google for the filename and it came up
```suggestion
      href: https://files.kff.org/attachment/Table-3-State-Adoption-of-Options-to-Cover-Immigrant-Populations-January-2021.pdf
```